backend/src/main/java/backend/mulkkam/cup/domain/vo/CupAmount.java

```diff
@@ -0,0 +1,19 @@
+package backend.mulkkam.cup.domain.vo;
+
+import static backend.mulkkam.common.exception.BadRequestErrorCode.INVALID_CUP_AMOUNT;
+
+import backend.mulkkam.common.exception.CommonException;
+
+public record CupAmount(
+        Integer value
+) {
```

Comment:
```suggestion
public record CupAmount(Integer value) {
```

고쳐주시면 감사하겟습니다 :) 컨벤션🤩